packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -12,312 +20,328 @@ import LoadingSpinner from '../../shared/components/LoadingSpinner';
-          notesQuery.refetch();
-        },
-        15000,
-        { leading: false, trailing: true }
-      ),
-    [language, notesQuery, word.id]
-  );
-  const { bookId, chapterNumber, verseNumber } = parseVerseId(verse.id);
-  const bdbCurrentVerseRef = `${
-    bdbBookRefNames[bookId - 1]
-  } ${chapterNumber}:${verseNumber}`;
+    useEffect(() => {
+      const { current } = lexiconEntryRef;
+      // Highlight references to the currently selected verse
+      current
+        ?.querySelectorAll(`a[data-ref="${bdbCurrentVerseRef}"]`)
+        .forEach((element) => element.classList.add('bg-yellow-300'));
+    }, [bdbCurrentVerseRef, lexiconEntry, t]);
 
-  const lexiconEntryRef = useRef<HTMLDivElement>(null);
-  const [previewElement, setPreviewElement] = useState<HTMLDivElement | null>(
-    null
-  );
-  const [previewVerseIds, setPreviewVerseIds] = useState<string[]>([]);
+    const openPreview = (anchorElement: HTMLAnchorElement) => {
+      const oldPreview = document.querySelector('#ref-preview');
+      oldPreview?.remove();
 
-  useEffect(() => {
-    const { current } = lexiconEntryRef;
-    // Highlight references to the currently selected verse
-    current
-      ?.querySelectorAll(`a[data-ref="${bdbCurrentVerseRef}"]`)
-      .forEach((element) => element.classList.add('bg-yellow-300'));
-  }, [bdbCurrentVerseRef, lexiconEntry, t]);
+      const reference = anchorElement.getAttribute('data-ref') ?? '';
+      setPreviewVerseIds(parseReferenceRange(reference, t));
 
-  const openPreview = (anchorElement: HTMLAnchorElement) => {
-    const oldPreview = document.querySelector('#ref-preview');
-    oldPreview?.remove();
+      const previewElement = document.createElement('div');
+      previewElement.id = 'ref-preview';
+      previewElement.style.width = 'calc(100% + 32px)';
+      previewElement.style.margin = '4px -16px';
+      previewElement.style.padding = '8px 16px';
+      previewElement.style.backgroundColor = '#ffffff80';
+      previewElement.style.float = 'left';
+      anchorElement.insertAdjacentElement('afterend', previewElement);
+      setPreviewElement(previewElement);
+    };
 
-    const reference = anchorElement.getAttribute('data-ref') ?? '';
-    setPreviewVerseIds(parseReferenceRange(reference, t));
+    const [tabIndex, setTabIndex] = useState(0);
 
-    const previewElement = document.createElement('div');
-    previewElement.id = 'ref-preview';
-    previewElement.style.width = 'calc(100% + 32px)';
-    previewElement.style.margin = '4px -16px';
-    previewElement.style.padding = '8px 16px';
-    previewElement.style.backgroundColor = '#ffffff80';
-    previewElement.style.float = 'left';
-    anchorElement.insertAdjacentElement('afterend', previewElement);
-    setPreviewElement(previewElement);
-  };
+    const [shouldFocusNotes, setShouldFocusNotes] = useState(false);
+    useImperativeHandle(ref, () => ({
+      openNotes: () => {
+        setTabIndex(1);
+        setShouldFocusNotes(true);
```

Comment:
If here we are calling focus on the rich text input ref, we may need to use setTimeout with no delay to force the focus call to occur after the browser has had a chance to repaint after we've switched the active tab